server/src/main/java/org/elasticsearch/search/lookup/SearchLookup.java

```diff
@@ -27,31 +27,22 @@
 import java.util.function.Function;
 
 public class SearchLookup {
-
-    final DocLookup docMap;
-
-    final SourceLookup sourceLookup;
```

Comment:
I think we still need `SourceLookup` here both for significant text, and for the thing we talked about where I wanted `SearchLookup` on `FieldMapper#lookup`.

Reply:
👍  will restore it.